lib/mesh.js

```diff
@@ -2162,11 +2155,12 @@ Mesh.prototype.getPackagedModules = function (config, callback) {
 
         var happnerFile = match.filename;
         var moduleName = match.moduleName;
+        var happnerConfig;
         // var version = match.version; // <---------- not yet used...
         // var parts = match.parts;
 
         try {
-          var happnerConfig = require(happnerFile);
+          happnerConfig = require(happnerFile);
         } catch(e) {
           return cb();
```

Comment:
should perhaps not fail silently

Reply:
added a logline and issue for this, I think this is linked to a whole bunch of functionality we dont use - which may be good to prune from the system